sk-usbhid.c

```diff
@@ -1050,7 +1113,6 @@ sk_sign(uint32_t alg, const uint8_t *data, size_t datalen,
 	response = NULL;
 	ret = 0;
  out:
-	explicit_bzero(message, sizeof(message));
```

Comment:
any reason why this is removed?

Reply:
It's dead code on Windows. I've re-added it under #ifndef WINDOWS in 843d4d79.